topostats/filters.py

```diff
@@ -242,6 +243,27 @@ def calc_gradient(self, array: np.ndarray, shape: int) -> np.ndarray:
         """Calculate the gradient of an array."""
         return self.calc_diff(array) / shape
 
+    def average_background(self, image: np.ndarray, mask: np.ndarray = None) -> np.ndarray:
+        """Zero the background by subtracting the non-masked mean from all pixels.
+
+        Parameters
+        ----------
+        image: np.array
+            Numpy array representing image.
+        mask: np.array
+            Mask of the array, should have the same dimensions as image.
+
+        Returns
+        -------
+        np.ndarray
+            Numpy array of image zero averaged.
+        """
+        if mask == None:
```

Comment:
I believe this should be `if mask is None:` ?

Reply:
Ah yes my bad! Thank you Sylvia!